setup.py

```diff
@@ -0,0 +1,188 @@
+        to make sure that this happens to be build/ relative to the current
+        working directory during the install phase since otherwise the install
+        step won't be able to find the assets that have been built by the build
+        step.
+        """
+        if "build" not in self.distribution.command_obj:
+            raise ValueError("There is no build directory since no build is being performed in this invocation of setup.py")
+
+        return self.distribution.command_obj["build"].build_base
+
+    @property
+    def abs_builddir(self):
+        r"""
+        Return the absolute path of the build directory, equivalent to @abs_builddir@ in automake.
+
+        The limitations of `builddir` apply to this property as well.
+        """
+        builddir = self.builddir
+        if not os.path.isabs(builddir):
+            builddir = os.path.join(self.abs_srcdir, builddir)
+
+        return builddir
+
+    @property
+    def destdir(self):
+        r"""
+        Return the installation prefix for this package in site-packages (or the user directory.)
+
+        This is the value that you want to pass to a configure's --prefix flag.
+        Note that naturally this value is only available when setup was asked
+        to install this package. In particular, this is not available when
+        trying to build a wheel.
+
+        As a consequence this value is also not available initially when
+        invoking `pip install` since that tries to build a wheel first. You
+        might want to invoke pip install with `--no-binary :all:` so that pip
+        skips to a regular install where this value is available.
+        """
+        if "install" not in self.distribution.command_obj:
+            raise ValueError("Cannot determine installation prefix in this build which does not install.")
+        return os.path.join(self.distribution.command_obj["install"].install_lib, self.distribution.get_name())
+
+    @property
+    def abs_srcdir(self):
+        r"""
+        Return the absolute path of the source directory, i.e., the directory where this setup.py is.
+
+        This is the equivalent to @abs_srcdir@ in automake.
+        """
+        return os.path.abspath(os.path.dirname(__file__) or ".")
+
+    @property
+    def MAKE(self):
+        r"""
+        Return the name of the make command which might have been overridden by
+        the MAKE environment variable.
+        """
+        return os.getenv("MAKE", "make")
+
+
+class NoBinaryBuild(build):
+    r"""
+    Disables building binary wheels for this package.
+
+    Since such binary wheels are not relocatable because we hard code library
+    paths into our hedaer files so cppyy known where things are located.
+    """
+
+    def run(self):
+        raise NotImplementedError("No binary wheels can be built for Antic currently because the installation prefix is hard-coded in some of its header files. To skip this step when using pip, run with --no-binary :all:")
```

Comment:
```suggestion
        raise NotImplementedError("No binary wheels can be built for Antic currently. To skip this step when using pip, run with --no-binary :all:")
```

Copy & paste error from the e-antic setup.